.github/workflows/coverage.yml

```diff
@@ -0,0 +1,34 @@
+on:
+  push:
+    branches:
+      - main
```

Comment:
Hmm, I personally like running code coverage checks for PRs as well (you can even have a bot add a comment with the coverage changes) but this of course isn't a must.

Reply:
I was worried that we'd reach the usage limits for our free plan since the code coverage step takes rather long (>6min), but [reviewing them again](https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration) it seems like it's well within our limits, so I'll adjust the PR.

Reply:
Done in 181eff42a48c74f9a002d3a6a79bc5c891a02861. It seems your old review became stale, so could you approve again?

Reply:
Awesome :) But couldn't we just add a `pull_request:` to this workflow to have it also run for PRs? That would avoid some code duplication.

Reply:
Ah, you already remove the other file, sorry! I'll approve the PR :)